Tests/OpenAPIKitTests/Schema Object/SchemaFragmentCombiningTests.swift

```diff
@@ -343,47 +340,62 @@ final class SchemaFragmentCombiningTests: XCTestCase {
     }
 
     func test_deeperObjectFragments() throws {
-        try assertOrderIndependentCombinedEqual(
-            [
-                .object(.init(), .init(properties: [:], additionalProperties: .init(true))),
-                .object(.init(description: "nested"), .init(properties: [:])),
-                .object(
-                    .init(),
-                    .init(
-                        properties: [
-                            "required": .string
-                        ],
-                        minProperties: 2
-                    )
-                ),
-                .object(
-                    .init(),
-                    .init(
-                        properties: [
-                            "optional": .boolean(required: false),
-                            "someObject": .object(required: false),
-                            "anything": .fragment(.init(description: nil))
-                        ],
-                        minProperties: 2
-                    )
+        let fragments: [JSONSchema] = [
+            .object(.init(), .init(properties: [:], additionalProperties: .init(true))),
+            .object(.init(description: "nested"), .init(properties: [:])),
+            .object(
+                .init(),
+                .init(
+                    properties: [
+                        "required": .string
+                    ],
+                    minProperties: 2
                 )
-            ],
+            ),
             .object(
-                .init(description: "nested"),
-                DereferencedJSONSchema.ObjectContext(
-                    .init(
-                        properties: [
-                            "required": .string,
-                            "optional": .boolean(required: false),
-                            "someObject": .object(required: false),
-                            "anything": .fragment(.init(description: nil))
-                        ],
-                        additionalProperties: .init(true),
-                        minProperties: 2
-                    )
-                )!
+                .init(),
+                .init(
+                    properties: [
+                        "optional": .boolean(required: false),
+                        "someObject": .object(required: false),
+                        "anything": .fragment(.init(description: nil))
+                    ],
+                    minProperties: 2
+                )
             )
-        )
+        ]
+        
+        try assertCombinedEqual(fragments, .object(
```

Comment:
These tests are now a little bit of a pain because for two ordered dictionaries to be equal, the order of keys must be equal. I'm surprised there weren't more tests that were failing because of that.